apps/backend/src/authn/authn.service.ts

```diff
@@ -26,6 +30,37 @@ export class AuthnService {
     }
   }
 
+  async validateOrCreateUser(
+    email: string,
+    firstName: string,
+    lastName: string
+  ): Promise<any> {
```

Comment:
This will always return a User object correct? Can the type be updated accordingly?